### PR TITLE
[ADD] - Added Project Parent module

### PR DESCRIPTION
**Pull request by YogeshMahera-SerpentCS**
- Added Project Parent Module

### CODE REVIEW

**Comment by elicoidal** [comment]
@pedrobaeza @dreispt your feeling? (we need it for Business Requirements)

File: project_parent/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```
Comment (elicoidal) [inline project_parent/__manifest__.py]: Add Elico copyright

File: project_parent/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+{
+    'name': 'Project Parent',
+    'summary': """
+        Project Patent.""",
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'Odoo Community Association (OCA)',
```
Comment (elicoidal) [inline project_parent/__manifest__.py]: Add Elico

File: project_parent/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    'website': 'https://www.github.com/OCA/account-analytic.git',
```
Comment (elicoidal) [inline project_parent/__manifest__.py]: adapt

File: project_parent/README.rst [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Eric Caudal<eric.caudal@elico-corp.com>
```
Comment (elicoidal) [inline project_parent/README.rst]: add space after the name

**Review by elicoidal — COMMENTED** [review]
No python at all?

**Comment by YogeshMahera-SerpentCS** [comment]
@elicoidal Could you please check it

File: project_parent/views/project_parent.xml [hunk 5]
```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.ui.view" id="projecet_parent_edit_project">
```
Comment (elicoidal) [inline project_parent/views/project_parent.xml]: Improve the id

File: project_parent/views/project_parent.xml [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+        <field name="inherit_id" ref="project.view_project"/>
+        <field name="priority">16</field>
+        <field name="arch" type="xml">
+            <field name="partner_id" position="after">
+                <field name="parent_id"/>
+            </field>
+        </field>
+   </record>
+
+    <record id="view_project_kanban_inherited_business_requirment" model="ir.ui.view">
```
Comment (elicoidal) [inline project_parent/views/project_parent.xml]: Improve the id (not anymore related to BR

**Review by elicoidal — CHANGES_REQUESTED** [review]
I modified the README

**Comment by YogeshMahera-SerpentCS** [comment]
@elicoidal Could you please review it

File: project_parent/views/project_parent.xml [hunk 7]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.ui.view" id="view_project_parent_edit_project">
```
Comment (elicoidal) [inline project_parent/views/project_parent.xml]: id then model

**Review by elicoidal — APPROVED** [review]

**Comment by YogeshMahera-SerpentCS** [comment]
@elicoidal AMENDED Changes

**Comment by elicoidal** [comment]
@pedrobaeza @dreispt: shall we proceed here? Shall we apply the changes directly in account_analytic_parent?

**Comment by pedrobaeza** [comment]
It can make sense to have a different project hierarchy than analytic hierarchy, as not all accounts are projects, and not all possible parent accounts are also projects.

Then, I think you can continue with this module.

File: project_parent/__manifest__.py [hunk 8]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2017 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Project Parent',
+    'summary': """
+        Project Patent.""",
```
Comment (pedrobaeza) [inline project_parent/__manifest__.py]: Don't put summary if it's the same than the name, and it contain a typo.

File: project_parent/__manifest__.py [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    "author": "Elico Corp, Odoo Community Association (OCA)",
+    'website': 'https://www.elico-corp.com/',
+    'depends': [
+        'account_analytic_parent',
```
Comment (pedrobaeza) [inline project_parent/__manifest__.py]: Don't depend on this one and create your own field parent_project_id as many2one reference to project.

File: project_parent/views/project_parent.xml [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+         <field name="name">project.project.select</field>
+         <field name="model">project.project</field>
+         <field name="inherit_id" ref="project.view_project_project_filter"/>
+         <field name="type">search</field>
+         <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="parent_id"/>
+            </field>
+           <xpath expr="//filter[contains(@domain, user_id)]" position="after">
+              <filter string="Parent" icon="terp-folder-orange"
```
Comment (pedrobaeza) [inline project_parent/views/project_parent.xml]: icon is deprecated

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
You need to change the approach of the module as outlined in the comment.

**Comment by YogeshMahera-SerpentCS** [comment]
@elicoidal @pedrobaeza Could you please review it

**Review by elicoidal — APPROVED** [review]
LGTM (functional tests)

File: project_parent/models/account_analytic_account.py [hunk 11]
```diff
@@ -0,0 +1,14 @@
...
+
+from odoo import fields, models
+
+
+class AccountAnalyticAccount(models.Model):
+    _inherit = 'account.analytic.account'
+
+    parent_project_id = fields.Many2one(
+        comodel_name='project.project',
+        string='Parent Analytic Account'
```
Comment (pedrobaeza) [inline project_parent/models/account_analytic_account.py]: Parent Project

**Comment by YogeshMahera-SerpentCS** [comment]
@pedrobaeza AMENDED Changes

**Review by pedrobaeza — APPROVED** [review]

**Comment by elicoidal** [comment]
Thanks @pedrobaeza

**Pull request merged**